monkey/monkey_island/cc/services/telemetry/processing/exploit.py

```diff
@@ -17,6 +18,7 @@
     edge = get_edge_by_scan_or_exploit_telemetry(telemetry_json)
     update_network_with_exploit(edge, telemetry_json)
     update_node_credentials_from_successful_attempts(edge, telemetry_json)
+    add_exploit_extracted_creds_to_config(telemetry_json)
```

Comment:
Don't we need to encrypt the credential first?

Reply:
Already happening.
The `ConfigService.creds_add_*()` calls take care of it (see [`monkey_island/cc/services/config.py`](https://github.com/guardicore/monkey/blob/develop/monkey/monkey_island/cc/services/config.py)).

Reply:
Are the telemetries added compatible with `update_node_credentials_from_successful_attempts(edge, telemetry_json)`? Does the credentials appear in the report as found in a certain IP? Either you need to encrypt credentials in telemetry json and add them to network node or this is unused code and we should delete it